mypy/checkexpr.py

```diff
@@ -204,7 +204,7 @@ def visit_call_expr(self, e: CallExpr, allow_none_return: bool = False) -> Type:
                         or isinstance(typ, NameExpr) and node and node.kind == nodes.TYPE_ALIAS):
                     self.msg.type_arguments_not_allowed(e)
         self.try_infer_partial_type(e)
-        callee_type = self.accept(e.callee)
+        callee_type = self.accept(e.callee, disallow_any=False)
```

Comment:
That means we should probably go through all stubs in typeshed and parameters of type `Any` to `object`. I can look into it. Just to confirm, it shouldn't break anything, right?

I see your point.
I still think we should not give an error if a user is calling a function that takes `Any`. Often, they can't modify the stubs of the function they are calling.
That being said, I'm okay with disallowing this and then seeing if it's a problem for the users.

How should we handle unresolved functions? (I imagine this would be the case when `--disallow-any=expr` is used alongside `--ignore-missing-imports`)